test/run

```diff
@@ -431,6 +434,15 @@ kill_test_application
 echo "Testing incremental build"
 test_incremental_build
 
+# test express webapp
+prepare express-webapp
+ct_s2i_build_as_df file://${test_dir}/test-express-webapp ${IMAGE_NAME} ${IMAGE_NAME}-testexpresswebapp ${s2i_args} $(ct_build_s2i_npm_variables)
```

Comment:
Docker image has to be named test-express-webapp.
Otherwise cycle and especially image_exists will not find the image.

  for image in "${IMAGE_NAME}"-{test{app,hw},express,express-webapp}; do
    image_exists "$image" || continue
    docker rmi -f "$image"
  done

won't work.
```suggestion
ct_s2i_build_as_df file://${test_dir}/test-express-webapp ${IMAGE_NAME} ${IMAGE_NAME}-testexpress-webapp ${s2i_args} $(ct_build_s2i_npm_variables)
```